src/main/java/org/greenbuttonalliance/gbaresourceserver/customer/model/TownDetail.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2022 Green Button Alliance, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.greenbuttonalliance.gbaresourceserver.customer.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Embeddable;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Embeddable
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class TownDetail {
+
+	@Column(name = "town_code")
+	private String code;
+
+	@Column(name = "town_section")
+	private String section;
+
+	@Column(name = "town_name")
+	private String name;
+
+	@Column(name = "town_county")
+	private String county;
+
+	@Column(name = "town_state_or_province")
+	private String stateOrProvince;
+
+	@Column(name = "town_country")
+	private String country;
```

Comment:
Similar explanation here for the `town_` prefix on column names as for the column prefix in `StreetDetail`

Reply:
Agreed and it repeats the street_ naming convention design.